activepieces/activepieces
# feat: added multiselect dropdown

## What does this PR do?

Makes a multiselect dropdown available as a property for pieces

## Type of change


## How should this be tested?
Have an action within a piece that's of this type, make sure refreshers work and values are saved as an array.
